v3_x/git-remote-gosh/src/blockchain/snapshot/load/diffs/iterator.rs

```diff
@@ -133,6 +133,10 @@ impl DiffMessagesIterator {
                     &file_path,
                 )
                 .await?;
+                let snapshot_contract =
+                    GoshContract::new(original_snapshot.clone(), crate::abi::SNAPSHOT);
+                let snapshot_is_active = snapshot_contract.is_active(client).await?;
+                tracing::trace!("snap={original_snapshot} is {}", if snapshot_is_active { "ACTIVE" } else { "NON_ACTIVE" });
```

Comment:
v3 version of this file has more logs than others, I suggest add this extra tracing strings to at least the last version